custom_components/powercalc/aliases.py

```diff
@@ -44,18 +44,15 @@
 MODEL_DIRECTORY_MAPPING = {
     "ikea": {
     },
-    "ledvance": {"4058075168572": "Tibea TW Z3"},
+    "ledvance": {
+    },
     "mueller-licht": {
-        "45327": "45318",
     },
-    "osram": {"AC03642": "CLA60 TW"},
+    "osram": {
+    },
     "signify": {
     },
     "yeelight": {
-        "color2": "YLDP06YL",
-        "ceiling10": "YLDL01YL",
-        "mono1": "YLDP01YL",
-        "strip6": "YLDD05YL",
         ### No profiles yet ###
```

Comment:
So, didn't know if it was worth removing these.

Reply:
Yes discussion topic is the right place for a list of yeelight models and discuss naming issues there.